hack/versionbump/main.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("Usage: versionbump major|minor|patch version")
+		os.Exit(1)
+	}
+	bumpType, rawVersion := os.Args[1], os.Args[2]
+	version, err := semver.NewVersion(rawVersion)
+	if err != nil {
+		fmt.Printf("Cannot convert version `%s`: %s\n", rawVersion, err)
+		os.Exit(1)
+	}
+	var newVerson semver.Version
```

Comment:
What is this being used for? Just to update the version list in crdbversions?  If so then this will work, we're still in a semver format even if we don't actually use the first two values quite the same.

Reply:
Yeah. Our calver uses the same format as semver, it just doesn't follow the semantics of major/minor upgrades.

Reply:
Actually, in this context this is about the operator version, which uses semver AFAIK.